### ISSUE TITLE
Refreshing token from server not working 

### ISSUE THREAD

**Issue by lhdmulder** [smart-on-fhir/client-js#82]
I am using fhirclient on a node server. Refreshing the token is not working as it seems that it is not providing the refresh endpoint with the correct Authorization Header (should be: Basic base64(clientid:clientsecret)).

Thanks for the work!

**Comment by kolkheang** [comment]
We are also unable to fresh the token, but due to CORS and the `credentials: include` option in the request [here](https://github.com/smart-on-fhir/client-js/blob/master/src/Client.ts#L962). 

```
Access to XMLHttpRequest at 'https://example-authorization-endpoint.com/protocols/oauth2/profiles/smart-v1/token' from origin 'http://origin-endpoint.com' has been blocked by CORS policy: The value of the 'Access-Control-Allow-Origin' header in the response must not be the wildcard '*' when the request's credentials mode is 'include'. The credentials mode of requests initiated by the XMLHttpRequest is controlled by the withCredentials attribute.
```

**Comment by vlad-ignatov** [comment]
Thanks @lhdmulder! There is new `dev` version including a fix for that. Can you give it a try (`npm i fhirclient@dev`) and see if it works for you?

**Comment by vlad-ignatov** [comment]
The fix is now released in `2.3.5`. Feel free to reopen if it does not solve the issue.
